master/buildbot/db/enginestrategy.py

```diff
@@ -76,6 +76,9 @@ def connect_listener_enable_fk(connection, record):
         # try to enable WAL logging
         if u.database:
             def connect_listener(connection, record):
+                # default for PY2, not anymore for PY3, and cause issue with VACUUM
```

Comment:
Change this comment to:
```
# Sets auto-commit mode, which was the default for PY2, but not anymore fore PY3.
# This causes and issue with VACUUM (which thinks it is in a transaction while it is not).
```

Also make sure to check with pylint, and add any words that pylint doesn't like to the magic list. :)